ftrip.io.catalog-service/ftrip.io.catalog-service/Persistence/SeederException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace ftrip.io.catalog_service.Persistence
+{
+    public class SeederException:Exception
```

Comment:
Space before and after :. Also, see code smell as it is good practice for Exceptions to be Serializable.